test/utils.ts

```diff
@@ -0,0 +1,46 @@
+import crypto from "crypto";
+import { mkdirSync, writeFileSync, rmSync } from "fs";
+import { resolve } from "path";
+
+const tempPath = resolve(__dirname, "../node_modules/.tmp/test-mocks");
```

Comment:
Use `node_modules/.tmp/test-mocks` folder to store temporary files for tests. Seems to work pretty well and is easily deletable by the user if needed.

Reply:
We delete the temporary files for tests anyway.